src/csharp/Microsoft.Spark/Utils/AssemblyLoader.cs

```diff
@@ -56,6 +57,19 @@ internal static string[] GetAssemblySearchPaths()
             searchPaths.Add(Directory.GetCurrentDirectory());
             searchPaths.Add(AppDomain.CurrentDomain.BaseDirectory);
 
+            FileInfo[] archives = new DirectoryInfo(Directory.GetCurrentDirectory())
+                .GetFiles("*.zip");
+            foreach (var archive in archives)
+            {
```

Comment:
Personally, rather than extracting all .zip files within the folder. I would prefer to extract only the applications .zip file if present.  I don't, however, know the project well enough to know if it is possible to access the application .zip parameter that's passed to DotNetRunner from C#.

Reply:
See #194 as an alternative way of potentially implementing this.